sled-diagnostics/src/queries.rs

```diff
@@ -25,7 +30,27 @@ const ZONEADM: &str = "/usr/sbin/zoneadm";
 pub const DEFAULT_TIMEOUT: Duration = Duration::from_secs(10);
 
 pub trait SledDiagnosticsCommandHttpOutput {
-    fn get_output(self) -> String;
+    fn get_output(self) -> SledDiagnosticsQueryOutput;
+}
+
+#[derive(Clone, Debug, JsonSchema, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum SledDiagnosticsQueryOutput {
+    Success {
+        /// The command and it's arguments.
```

Comment:
```suggestion
        /// The command and its arguments.
```

Reply:
Fixed in [7b67092](https://github.com/oxidecomputer/omicron/pull/7228/commits/7b670922773578551239cf42a620521bc3eafc27)